apps/playwright-browser-tunnel/tests/demo.spec.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { test } from './testFixture';
+import { expect } from '@playwright/test';
+
+test('woohoo!', async ({ page }) => {
+  await page.goto('https://playwright.dev/');
```

Comment:
We should ideally have the demo test work even if you don't have internet and are just remoting into a local container.